ext/session/session.c

```diff
@@ -2049,14 +2065,14 @@ PHP_FUNCTION(session_set_save_handler)
 			ZEND_ASSERT(result == SUCCESS);
 
 			/* add shutdown function, removing the old one if it exists */
-			if (!register_user_shutdown_function("session_shutdown", sizeof("session_shutdown") - 1, &shutdown_function_entry)) {
+			if (!register_user_shutdown_function("session_shutdown", strlen("session_shutdown"), &shutdown_function_entry)) {
```

Comment:
There is no need to change `strlen()` to `sizeof`; any somewhat reasonable compiler is supposed to optimize `strlen` for literals.

Reply:
I went the other way, from sizeof to strlen

Reply:
D'oh! Makes sense, but maybe we should document that preference in [our coding style guidelines](https://github.com/php/php-src/blob/master/CODING_STANDARDS.md).